docs/reference/msbuild-targets.md

```diff
@@ -464,6 +465,41 @@ msbuild -t:restore -p:RestorePackagesConfig=true
 > [!NOTE]
 > `packages.config` restore is only available with `MSBuild 16.5+`, and not with `dotnet.exe`
 
+### Restoring with MSBuild static graph evaluation
+
+> [!NOTE]
+> With MSBuild 16.6+, NuGet has added an experimental feature to use static graph evaluation from the commandline that significantly decreases the restore time for large repositories.
+
+```cli
+msbuild -t:restore -p:RestoreUseStaticGraphEvaluation=true
+```
+
+Alternatively you can enable it by setting the property in a Directory.Build.Props.
+
+```xml
+<Project>
+  <PropertyGroup>
+    <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+  </PropertyGroup>
+</Project>
+```
+
+> [!NOTE]
+> As of Visual Studio 2019.x and NuGet 5.x, this feature is considered experimental and opt-in. Follow [NuGet/Home#9803](https://github.com/NuGet/Home/issues/9803) for details on when this feature will be enabled by default.
+
+Static graph restore changes the msbuild part of restore, the project reading and evaluation, but not the restore algorithm! The restore algorithm is the same across all NuGet tools (NuGet.exe, MSBuild.exe, dotnet.exe and Visual Studio).
+
+In very few scenarios, static graph restore may behave differently from current restore and certain declared PackageReferences or ProjectReferences might be missing.
```

Comment:
Do we have a list of these scenarios? Is it worth including here?

Reply:
There's been a few bugs here and there, but we've been fixing those issues.